linux-ubuntu-2404.pkr.js/linux-ubuntu-2404.Dockerfile

```diff
@@ -1,14 +1,14 @@
 ARG UBUNTU_24_04="ubuntu@sha256:04f510bf1f2528604dc2ff46b517dbdbb85c262d62eacc4aa4d3629783036096"
 FROM ${UBUNTU_24_04}
 
-ENV TIPI_DISTRO_MODE=default
+ENV TIPI_DISTRO_MODE=all
```

Comment:
As we now have a full one, do we want to put a default toolchain file with useful compiler defaults ?

Reply:
done